### DISCUSSION TITLE
Competitive Play

### DISCUSSION THREAD

**Discussion by mlutze** [SSoelvsten/wordrow#169] [General]
By setting the seed, multiple people can play against each other - the one with the highest score wins! Everyone can load the webpage, sit on a waiting screen until the competition starts.

**Comment by ssoelvsten** [comment]
Interesting! One could do so by providing the settings as part of the URL, i.e. seed, language, and game type. This also provides one the ability to share a certain game, challenging friends to beat one's high score on a specific "run".
